guides/cloud/aws_certification_advice.md

```diff
@@ -0,0 +1,105 @@
+# AWS Certification Advice
+
+## Purpose of this guide
+
+Made Tech are an [AWS Advanced Consulting Partner](https://partners.amazonaws.com/partners/001E0000016ppWOIAY/Made%20Tech) and encourage both technical and non-technical staff to undertake AWS Training and Certifications. This guide will detail some training paths and associated costs.
+
+AWS provides some non-technical training via the AWS Partner Portal. Guidance on how to create an account and take advantage of this training can be found [here](aws_partner_certs.md). 
+
+## General
+
+A guide on how to create an AWS Certification account and link it to your Made Tech AWS Partner Account is located [here](aws_partner_certs.md#aws-certification-account-creation)
+
+All AWS Exams can be taken remotely from home with either PSI or Pearson Vue.
+
+## Certification Costs
+
+Booking an AWS exam costs vary depending on the level of the exam.
+
+| Exam | Cost | Cost inc tax |
+| --- | --- | --- |
+| AWS Cloud Practitioner | $100 | $120 |
+| AWS Associate Level (Developer, SysOps, Architect) | $150 | $180 |
+| AWS Professional Level (DevOps Engineer, Architect) | $300 | $360 |
+| AWS Speciality Exams (Security, Networking, Database, ML, Data) | $300 | $360 |
+
+Each time you pass an AWS Exam you receive a [free practice exam and 50% off voucher](https://www.certmetrics.com/amazon/candidate/benefit_summary.aspx) for the next exam.
+
+This means an individual could achieve all 11 AWS Certifications for $1650!
```

Comment:
Is this good or bad? I can't tell.

Reply:
I think it's pretty good, some one day training courses cost that